cdm/core/src/main/java/ucar/nc2/internal/iosp/hdf5/H5headerNew.java

```diff
@@ -1601,7 +1649,7 @@ public class Vinfo {
 
     // chunked stuff
     boolean isChunked;
-    DataBTree btree; // only if isChunked
+    public DataBTree btree; // only if isChunked
```

Comment:
this doesn't need to be public if we're adding a getter and don't want it to be publicly settable

Reply:
Done.